### DISCUSSION TITLE
Accessibility to my project

### DISCUSSION THREAD

**Discussion by white10146** [community/community#42096] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

All my project is not working good .. I needed a help seriously please

**Comment by getlost01** [comment]
Provide the full description of your projects for better help.

**Reply by white10146** [reply]
What are the details needed

**Comment by ghostinhershell** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

Also, please be explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by 0768993338** [comment]
Thanks ive received the message.
